daemon/resize.go

```diff
@@ -37,5 +37,8 @@ func (daemon *Daemon) ContainerExecResize(name string, height, width int) error
 	if err != nil {
 		return err
 	}
+	if err = ec.WaitForReady(); err != nil {
```

Comment:
just do the select directly here since it's not used anywhere else. Otherwise `HasStarted` is a better name.

Reply:
copy that~